### ISSUE TITLE
is this bannable?

### ISSUE THREAD

**Issue by ShoelessTom** [hitrov/oci-arm-host-capacity#108]
is this bannable?

**Comment by hitrov** [comment]
Running every minute is probably is. 5 minutes period should be fine

**Comment by NanoG6** [comment]
Sorry if it's out of topic, but anyone success to get one in Seoul? I have the script running since July 2023 but still no luck..

**Comment by hitrov** [comment]
Man, just upgrade to PaYG and enjoy all the free stuff ;)

**Comment by NanoG6** [comment]
Yeah I thought so, any tips so I don't get any surprises?

**Comment by hitrov** [comment]
I don’t think so https://www.google.com/search?q=oci+pay+as+you+go+reddit just be cautious
